recipes/libsemigroups-python-bindings/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "libsemigroups-python-bindings" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "8a42bc0d26c2e1d8bc34d1ce6a71901c46b16e11f42ad56d77bbf5628f6364ad" %}
+{% set repo = "https://github.com/james-d-mitchell/libsemigroups" %}
+{% set home = "https://james-d-mitchell.github.io/libsemigroups/" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The python bindings are not yet merged in master; fetch them from their git branch instead
+  git_url: {{ repo }}.git
+  git_tag: python-bindings
+  #url: {{ repo }}/releases/download/v{{ version }}/libsemigroups-{{ version }}.tar.gz
+  #sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: cd python-bindings; python setup.py install
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - cython
+    - libsemigroups
```

Comment:
Can you add a pinning for this? Should the version of Python wrappers match libsemigroups version exactly?